src/lib.rs

```diff
@@ -1655,4 +1701,43 @@ mod tests {
         v.push(true);
         assert_eq!(v.vec.len(), 1);
     }
+
+    #[test]
+    fn to_bytes_1_byte_a() {
```

Comment:
To  match the names of other tests, this should be something like `test_to_bytes_single` and this should be merged with `to_bytes_1_byte_b`.

Reply:
Actually, `to_bytes_1_byte_b` is *exactly* the same test, so we should delete that one.